### ISSUE TITLE
Tags for 24.5.x removed

### ISSUE THREAD

**Issue by dlouzan** [getsentry/self-hosted#3172]
### Self-Hosted Version

-

### CPU Architecture

-

### Docker Version

-

### Docker Compose Version

-

### Steps to Reproduce

We deployed 24.5.0 last month and we just realized that the tags are gone. Was this intended? The changelog still lists them though: https://github.com/getsentry/self-hosted/blob/master/CHANGELOG.md

### Expected Result

Tags for 24.5.0 and 24.5.1 are available.

### Actual Result

No tags available: https://github.com/getsentry/self-hosted/tags

**Comment by hubertdeng123** [comment]
Yes, this was intentional. See here for more details:
https://github.com/getsentry/self-hosted/issues/3167

**Comment by liukch** [comment]
> Yes, this was intentional. See here for more details: #3167

No!!!! Please do not do that. A lot of people use these version in prod env. What if they meet some accident, where for them to find out these code? 

This is irreversible. The correct approach is to add some comments in a prominent position for version 24.6.x.

**Comment by max-wittig** [comment]
I agree with @liukch. We had to run `REINDEX index sentry_groupedmessage_view;` for the DB upgrade to succeed and after reading the comments in https://github.com/getsentry/self-hosted/issues/3167, I'm not sure that this was enough.

I know this is free software and we cannot expect anything, but this is not a way to handle upgrades.

**Comment by dlouzan** [comment]
Well, I agree with the other posters that removing git tags is a pretty weird move, but I guess since there's no packages you have no way to "yank" a version. We'll keep an eye on the linked issues. Thanks!
